system/Database/MigrationRunner.php

```diff
@@ -415,9 +415,17 @@ protected function getMigrationName($migration)
 	 */
 	protected function getVersion($group = 'default')
 	{
+		if (empty($group))
```

Comment:
In both of these methods I'd rather throw an exception here. Since this is a developer-oriented feature we shouldn't mask the problem, as that might cause someone to bang their head in frustration that something isn't working as expected. I'd rather throw the exception so they know what's wrong before it ever makes it to production.